catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -473,12 +473,12 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.64</minimum>
+                                            <minimum>0.62</minimum>
```

Comment:
If it makes you feel better, this didn't drop because I added code to this module and then didn't write tests for it, but rather that I pull code out of the module which was being tested. My next commit will include the update to the `SecurityPlugin` coverage which is significantly higher (partially as a result of the code I added).